www-apps/hugo/hugo-0.62.0.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit go-module bash-completion-r1
+
+# The fork with prefetched vendor packages using `go mod vendor`
+EGO_PN="github.com/g4s8/hugo"
+GIT_COMMIT="6608f1557054a7d0230dff260c1b66bc19e65ec8"
+KEYWORDS="~amd64"
+
+DESCRIPTION="The world's fastest framework for building websites"
+HOMEPAGE="https://gohugo.io https://github.com/gohugoio/hugo"
+SRC_URI="https://${EGO_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="Apache-2.0 Unlicense BSD BSD-2 MPL-2.0"
+SLOT="0"
+IUSE="+sass +bash-completion doc"
+
+RESTRICT="test"
```

Comment:
In future, please add a comment above saying why tests are restricted, and preferably a link to upstream issue about it.